reconcile/terraform_resources.py

```diff
@@ -335,7 +347,11 @@
     }
   }
 }
-""" % (indent(TF_RESOURCE, 6*' '))
+""" % (
+    indent(Template(TF_RESOURCE_AWS).render(account=False), 6*' '),
+    indent(Template(TF_RESOURCE_AWS).render(account=True), 6*' ')
+)
+
```

Comment:
... even if it means investigating how to deal with nested templates. Or refactoring them.

Reply:
especially if it means to to rabbit hole into this topic as part of this PR?